resources/functionMap.php

```diff
@@ -11696,7 +11696,7 @@
 'str_replace' => ['string|array', 'search'=>'string|array', 'replace'=>'string|array', 'subject'=>'string|array', '&w_replace_count='=>'int'],
 'str_rot13' => ['string', 'str'=>'string'],
 'str_shuffle' => ['string', 'str'=>'string'],
-'str_split' => ['array<int,string>|false', 'str'=>'string', 'split_length='=>'int'],
+'str_split' => ['array<int,string>|false', 'str'=>'string', 'split_length='=>'positive-int'],
```

Comment:
`str_split` always returns a non-empty array.. I tried adding this here, but it seems the `&nonEmpty` signature doesn't work from this file

Reply:
since maybe controversial, put it into a separate PR:
https://github.com/phpstan/phpstan-src/pull/632

Reply:
The correct is `'non-empty-array<int,string>|false'`

Reply:
ahh makes sense.. https://github.com/phpstan/phpstan-src/pull/633